packages/lesspass-pure/src/components/Options.vue

```diff
@@ -115,8 +115,16 @@
 
   export default {
     name: 'options',
-    computed: {
-      ...mapState(['password']),
+    props: {
+      password: {
+        type: Object,
+        required: true
+      },
```

Comment:
I need to parametrize the state to update when updating the options, as we want to update the defaultPassword in the options page and the current password otherwise. I don't know if passing the state as prop as I did is the right way to do that in Vue. Otherwise I could just pass a string containing the key of the password object and let the component retrieve the corresponding password. Tell me if you think it would be better!